// Uncaught SyntaxError due to "age": B
SyntaxError: Unexpected token 'B', ..."
  "age": B
}" is not valid JSON
    at JSON.parse (<anonymous>)
    at parse (C:\Users\saswa\Node_Project\src\node_modules\body-parser\lib\types\json.js:92:19)
    at C:\Users\saswa\Node_Project\src\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:211:14)
    at invokeCallback (C:\Users\saswa\Node_Project\src\node_modules\raw-body\index.js:238:16)
    at done (C:\Users\saswa\Node_Project\src\node_modules\raw-body\index.js:227:7)
    at IncomingMessage.onEnd (C:\Users\saswa\Node_Project\src\node_modules\raw-body\index.js:287:7)
    at IncomingMessage.emit (node:events:513:28)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)


// Caught SyntaxError due to "age": B
{
  "status": 400,
  "message": "Unexpected token 'B', ...\"\n  \"age\": B\n}\" is not valid JSON",
  "name": "SyntaxError",
  "type": "entity.parse.failed",
  "error_object": {
    "expose": true,
    "statusCode": 400,
    "status": 400,
    "body": "{\n  \"username\":\"MNO\",\n  \"age\": B\n}",
    "type": "entity.parse.failed"
  },
  "error_stack": "SyntaxError: Unexpected token 'B', ...\"\n  \"age\": B\n}\" is not valid JSON\n    at JSON.parse (<anonymous>)\n    at parse (C:\\Users\\saswa\\Node_Project\\src\\node_modules\\body-parser\\lib\\types\\json.js:92:19)\n    at C:\\Users\\saswa\\Node_Project\\src\\node_modules\\body-parser\\lib\\read.js:128:18\n    at AsyncResource.runInAsyncScope (node:async_hooks:211:14)\n    at invokeCallback (C:\\Users\\saswa\\Node_Project\\src\\node_modules\\raw-body\\index.js:238:16)\n    at done (C:\\Users\\saswa\\Node_Project\\src\\node_modules\\raw-body\\index.js:227:7)\n    at IncomingMessage.onEnd (C:\\Users\\saswa\\Node_Project\\src\\node_modules\\raw-body\\index.js:287:7)\n    at IncomingMessage.emit (node:events:513:28)\n    at endReadableNT (node:internal/streams/readable:1696:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)"
}


// Database Connection Error run from try catch block w/o error object.
{
  status: undefined,
  message: 'querySrv ECONNREFUSED _mongodb._tcp.firstnodejsappdb.h82ik.mongodb.net',
  name: 'Error',
  errno: undefined,
  code: 'ECONNREFUSED',
  syscall: 'querySrv',
  hostname: '_mongodb._tcp.firstnodejsappdb.h82ik.mongodb.net',
  error_stack: 'Error: querySrv ECONNREFUSED _mongodb._tcp.firstnodejsappdb.h82ik.mongodb.net\n' +
    '    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:293:17)'
}

// Uncaught Database Connection Error 
node:internal/dns/promises:293
    this.reject(new DNSException(err, this.bindingName, this.hostname));
                ^

Error: querySrv ECONNREFUSED _mongodb._tcp.firstnodejsappdb.h82ik.mongodb.net
    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:293:17) {
  errno: undefined,
  code: 'ECONNREFUSED',
  syscall: 'querySrv',
  hostname: '_mongodb._tcp.firstnodejsappdb.h82ik.mongodb.net'
}

// Caught database connection error
Could not connect to the database.
{
  status: undefined,
  message: 'querySrv ECONNREFUSED _mongodb._tcp.firstnodejsappdb.h82ik.mongodb.net',
  name: 'Error',
  errno: undefined,
  code: 'ECONNREFUSED',
  type: undefined,
  syscall: 'querySrv',
  hostname: '_mongodb._tcp.firstnodejsappdb.h82ik.mongodb.net',
  error_stack: 'Error: querySrv ECONNREFUSED _mongodb._tcp.firstnodejsappdb.h82ik.mongodb.net\n' +
    '    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:293:17)'
}